src/openstudio_lib/WaterUseEquipmentInspectorView.cpp

```diff
@@ -260,7 +260,9 @@ WaterUseEquipmentDefinitionInspectorView::WaterUseEquipmentDefinitionInspectorVi
   mainGridLayout->addWidget(label, 6, 0);
 
   m_targetTemperatureScheduleDZ = new OSDropZone(m_targetTemperatureScheduleVC);
+  m_targetTemperatureScheduleDZ->setMinItems(0);
   m_targetTemperatureScheduleDZ->setMaxItems(1);
+  m_targetTemperatureScheduleDZ->setItemsAcceptDrops(true);
```

Comment:
Configure all the drop zones to accept item drops. Fixes @Ski90Moo 's comment at https://github.com/openstudiocoalition/OpenStudioApplication/issues/683#issuecomment-2006775704

Demo:

![683_draggable](https://github.com/openstudiocoalition/OpenStudioApplication/assets/5479063/37d6370e-abf7-4bab-bbb9-b9b9e7d0cb98)